### PR TITLE
Rey Tests, Padding change on sidenav

### PR DESCRIPTION
**Pull request by Shinku24**
Created Rey test module
Changed padding from sidenav to every test individually

### CODE REVIEW

File: psique/src/App/test/king/king.js [hunk 1]
```diff
@@ -1,8 +1,576 @@
...
+              <DialogContentText>ID: {patientId}</DialogContentText>
+              <br></br>
+              <DialogContentText> Fecha: {testDate} </DialogContentText>
+              <DialogContentText >Tiempo (segundos): {time}</DialogContentText>
+              <DialogContentText >Puntaje: {score}</DialogContentText>
+              <br></br>
+              <DialogContent>
+                <Grid container justify="center" spacing={2}>
+                  <Grid item xs={4}> <Link to={'/home'}><Button variant="contained" color="primary">Sí</Button></Link> </Grid>
+                  <Grid item xs={4}> <Button variant="contained" color="secondary" onClick={() => {setModalControl(false)}}>No</Button> </Grid>
```
Comment (Daniel0205) [inline psique/src/App/test/king/king.js:268]: use CustomButton component

File: psique/src/App/test/king/king.js [hunk 2]
```diff
@@ -1,8 +1,576 @@
...
+          <br></br>
+          <Grid container justify="center" spacing={2}>
+            <Grid item> <FormLabel> Tiempo: </FormLabel> </Grid>
+            <Grid item> <TextField required label="Minutos" value={minutes} onChange={(e)=>setMinutes(e.target.value)} maxLength="3" inputProps={{ 'aria-label': 'minutes' }} /> </Grid>
+            <Grid item> <TextField required label="Segundos" value={seconds}  onChange={(e)=>setSeconds(e.target.value)} maxLength="2" inputProps={{ 'aria-label': 'seconds' }} /> </Grid>
+          </Grid>
+          <br></br>
+
+          <Grid container justify="center">
+            <TextField  id="copyType" multiline label="Tipo de Copia" onChange={(e)=>setCopyType(e.target.value)} value={copyType} inputProps={{ 'aria-label': 'copy-type' }}/>
```
Comment (Daniel0205) [inline psique/src/App/test/king/king.js:321]: U can use a combobox to limit the possible answers

File: psique/src/App/test/king/king.js [hunk 3]
```diff
@@ -1,8 +1,576 @@
...
+                shrink: true,
+              }}
+            />
+          </Grid>
+
+          <br></br>
+          <Grid container justify="center" spacing={2}>
+            <Grid item> <FormLabel> Tiempo: </FormLabel> </Grid>
+            <Grid item> <TextField required label="Minutos" value={minutes} onChange={(e)=>setMinutes(e.target.value)} maxLength="3" inputProps={{ 'aria-label': 'minutes' }} /> </Grid>
+            <Grid item> <TextField required label="Segundos" value={seconds}  onChange={(e)=>setSeconds(e.target.value)} maxLength="2" inputProps={{ 'aria-label': 'seconds' }} /> </Grid>
```
Comment (Daniel0205) [inline psique/src/App/test/king/king.js:316]: change the type of the TextField to number

File: psique/src/App/test/king/king.js (hunk 2 above)
Comment (Shinku24) [reply psique/src/App/test/king/king.js:321]: Are you refering to copyType textfield? I leave it open for specialists to write, even if the guide give some pointers, it looks like it more free to write observations of the specific drawn figure of the patient

Comment (Daniel0205) [reply psique/src/App/test/king/king.js:321]: Ohh mai gash, it's ok so v:

File: psique/src/App/test/king/king.js [hunk 4]
```diff
@@ -1,8 +1,570 @@
...
+import Paper from '@material-ui/core/Paper';
+import ErrorIcon from '@material-ui/icons/Error';
+import Divider from '@material-ui/core/Divider';
+import {Link} from "react-router-dom";
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogActions from '@material-ui/core/DialogActions';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import moment from "moment";
```
Comment (Daniel0205) [inline psique/src/App/test/king/king.js:20]: Module not found

**Pull request merged**